UI/src/application.js

```diff
@@ -23,7 +23,7 @@
     });
 
     var pointAssets = _.map(pointAssetSpecs, function(spec) {
-      var collection = _.isUndefined(spec.collection ) ?  new PointAssetsCollection(backend, spec.layerName, spec.allowComplementaryLinks) : new spec.collection(backend, spec.layerName, spec.allowComplementaryLinks) ;
+      var collection = _.isUndefined(spec.collection ) ?  new PointAssetsCollection(backend, spec.layerName, spec.allowComplementaryLinks, verificationCollection, spec.typeId) : new spec.collection(backend, spec.layerName, spec.allowComplementaryLinks, verificationCollection) ;

```

Comment:
Probably we can just pass through the spec, instead of separate every variable. What I mean is: 

`new PointAssetsCollection(backend, spec, verificationCollection) `